openmeter/productcatalog/plan/adapter/phase.go

```diff
@@ -362,6 +363,11 @@ func (a *adapter) UpdatePhase(ctx context.Context, params plan.UpdatePhaseInput)
 				p.StartAfter = *params.StartAfter
 			}
 
+			if params.Discounts != nil {
+				query = query.SetDiscounts(*params.Discounts)
+				p.StartAfter = *params.StartAfter
```

Comment:
this looks out of place at first sight... can you help me understand?

Reply:
valid

Reply:
Good catch. Will fix.